packages/teams-js/src/public/interfaces.ts

```diff
@@ -368,154 +421,204 @@ export interface Context {
   userObjectId?: string;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.team.isArchived} instead
+   *
    * Indicates whether team is archived.
    * Apps should use this as a signal to prevent any changes to content associated with archived teams.
    */
   isTeamArchived?: boolean;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.host.name} instead
+   *
    * The name of the host client. Possible values are: Office, Orange, Outlook, Teams
    */
   hostName?: HostName;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.host.clientType} instead
+   *
    * The type of the host client. Possible values are : android, ios, web, desktop, rigel(deprecated, use teamsRoomsWindows instead),
    * surfaceHub, teamsRoomsWindows, teamsRoomsAndroid, teamsPhones, teamsDisplays
    */
   hostClientType?: HostClientType;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.page.frameContext} instead
+   *
    * The context where tab url is loaded (content, task, setting, remove, sidePanel)
    */
   frameContext?: FrameContexts;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.sharepoint} instead
+   *
    * SharePoint context. This is only available when hosted in SharePoint.
    */
   sharepoint?: any;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.tenant.teamsSku} instead
```

Comment:
This should be app.Context.user.tenant.teamsSku.

"user" is missing in the link.